### PR TITLE
update the intellij docs

### PR DESCRIPTION
**Pull request by devoncarew**
- add a link to setting up IntelliJ to the sidebar nav
- rename the sidebar nav section `Test, deploy, workflows` => `Development and tools`, and re-order to items a bit
- update the list of IntelliJ based tools

@mit-mit @sethladd

### CODE REVIEW

**Comment by sethladd** [comment]
deferring to @mit-mit

File: _includes/sidebar.html [hunk 1]
```diff
@@ -40,17 +40,16 @@
       <li><a href="https://codelabs.developers.google.com/codelabs/flutter-firebase/index.html#0">Firebase for Flutter - Codelab</a></li>
     </ul>
 
-
-  <li class="sidebar-title">Test, deploy, workflows</li>
+  <li class="sidebar-title">Development and tools</li>
 
     <ul class="sidebar-items">
+      <li><a href="/intellij-ide/">Using IntelliJ</a></li>
```
Comment (mit-mit) [inline _includes/sidebar.html]: Actually, now this also applies to Android Studio? Maybe reword to something like 'Flutter IDE support'

File: intellij-setup.md [hunk 2]
```diff
@@ -13,18 +13,18 @@ This page describes how to set up an IntelliJ IDE to develop Flutter apps.
...
 
 ### Installing IntelliJ
 
-You can use the IntelliJ plug-ins with one of the following JetBrains IDEs:
+You can use the IntelliJ plugin with the following JetBrains IDEs:
 
-* [IntelliJ IDEA](https://www.jetbrains.com/idea/download/), Community (free) edition, version 2017.1 or later.
-* [IntelliJ IDEA](https://www.jetbrains.com/idea/download/), Ultimate edition, version 2017.1 or later.
-* [IntelliJ WebStorm](https://www.jetbrains.com/webstorm/download), version 2017.1 or later.
+* [IntelliJ CE](https://www.jetbrains.com/idea/download/) Community Edition (CE), version 2017.1 or later.
```
Comment (mit-mit) [inline intellij-setup.md]: Don't we require 2017.2 by now?

File: _includes/sidebar.html (hunk 1 above)
Comment (devoncarew) [reply _includes/sidebar.html]: Makes sense - updated to `Using Flutter IDEs`

File: intellij-setup.md (hunk 2 above)
Comment (devoncarew) [reply intellij-setup.md]: We support 2017.1 for internal IntelliJ; Android Studio 3.0 is also based on it. We could update to say that more clearly (2017.1 is the min, 2017.2 works and is recommended, 2017.3 coming shortly, ...)

Comment (mit-mit) [reply intellij-setup.md]: I think the current is fine then
